content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
```

Comment:
Please double-check these with the [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md) document.